doc/maths.md

```diff
@@ -506,6 +506,22 @@ Let `x` be a tensor and `n` a number.
 
 In this section, we explain basic mathematical operations for Tensors.
 
+<a name="torch.Tensor.equal"></a>
+### [boolean] equal([tensor1,] tensor2) ###
+
+Returns `true` iff the values of `tensor1` and `tensor2` are exactly the same.
```

Comment:
Mention that this is much more efficient than `a:eq(b):all()`, as it avoids allocation of a temporary and can short circuit.